packages/block-editor/src/components/font-appearance-control/index.js

```diff
@@ -97,7 +97,6 @@ export default function FontAppearanceControl( props ) {
 		name: __( 'Default' ),
 		style: { fontStyle: undefined, fontWeight: undefined },
 	};
-	debugger;
```

Comment:
These debuggers were part of the previous commit. I'll square before merging to clean this up -- wanted to keep it as reference in case I need to go back and debug again.